apps/hendelse-api/src/test/java/no/nav/registre/testnorge/hendelse/provider/HendelseControllerIntegrationTest.java

```diff
@@ -0,0 +1,78 @@
+package no.nav.registre.testnorge.hendelse.provider;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.sql.Date;
+import java.time.LocalDate;
+
+import no.nav.registre.testnorge.dto.hendelse.v1.HendelseType;
+import no.nav.registre.testnorge.dto.hodejegeren.v1.PersondataDTO;
+import no.nav.registre.testnorge.hendelse.domain.Hendelse;
+import no.nav.registre.testnorge.hendelse.repository.model.HendelseModel;
+import no.nav.registre.testnorge.test.JsonWiremockHelper;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureWireMock(port = 0)
+@AutoConfigureMockMvc
+@TestPropertySource(locations = "classpath:application-test.properties")
+class HendelseControllerIntegrationTest {
+
+    @Autowired
+    private MockMvc mvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Test
+    void should_get_hendelser() throws Exception {
+
+        String personIdent = "12125678903";
+
+
+        JsonWiremockHelper
+                .builder(objectMapper)
+                .withUrlPathMatching("(.*)/v1/persondata")
+                .withQueryParam("ident", personIdent)
+                .withResponseBody(PersondataDTO
+                        .builder()
+                        .fnr(personIdent)
+                        .fornavn("Hans")
+                        .etternavn("Hansen")
+                        .build())
+                .stubGet();
+
+        JsonWiremockHelper
+                .builder(objectMapper)
+                .withUrlPathMatching("(.*)/v1/hendelser")
+                .withResponseBody(new Hendelse(new HendelseModel(21321L, HendelseType.SYKEMELDING_OPPRETTET, Date.valueOf(LocalDate.now()), Date.valueOf(LocalDate.now()), Date.valueOf(LocalDate.now()), personIdent)))
+                .stubGet();
+
+        String json = mvc.perform(get("/api/v1/hendelser")
+                .contentType(MediaType.APPLICATION_JSON))
+                .andReturn()
+                .getResponse()
+                .getContentAsString();
+
+        //TODO fullføre test, returnerer tom json
```

Comment:
Tror det er like greit å fjerne hele denne klassen. Har sett på det og det ser ut som vi faktisk må legge til en hendelse i repository for å få lagd en integrasjonstest. Så jeg tror det er bedre å vente til Martin er tilbake så vi kan sjekke med han, siden han som har skrevet koden og han har mer erfaring med integrasjonstester generelt.

Reply:
Ahh okay, såpass 😅 Ja det høres ut som en god ide, fjerner den 👍